MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,121 @@
 import UIKit
 
-final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
-    override func viewDidLoad() {
+final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
+    func didReceiveNextQuestion(question: QuizQuestion?) {
+        guard let question = question else {return}
+        currentQuestion = question
+        let viewModel = convert(model: question)
+        DispatchQueue.main.async { [weak self] in
+            self?.show(quiz: viewModel)
+        }}
+    
+    override func viewDidLoad()
+    {
```

Comment:
Фигурная скобка убежала 
```suggestion
    
    override func viewDidLoad() {
```